tests/unit/checkboxradio/checkboxradio_options.js

```diff
@@ -0,0 +1,224 @@
+/*
+* checkboxradio_methods.js
+*/
+
+(function($) {
+
+module( "Checkboxradio: options" );
+
+function assertDisabled( widget, checkbox ) {
+	QUnit.assert.hasClasses( widget, "ui-state-disabled",
+		"label gets ui-state-disabled" );
+	strictEqual( checkbox.is( ":disabled" ), true,
+		"checkbox is disabled" );
+}
+
+function assertEnabled( widget, checkbox ) {
+	QUnit.assert.lacksClasses(  widget, "ui-state-disabled",
+		"label has ui-state-disabled removed when disabled set to false" );
+	strictEqual( checkbox.is( ":disabled" ), false,
+		"checkbox has disabled prop removed when disabled set to false" );
+}
+
+test( "disabled", function() {
+	var checkbox = $( "#checkbox-option-disabled" ),
+		widget;
+	expect( 6 );
+	checkbox.checkboxradio({
+		disabled: true
+	});
+
+	widget = checkbox.checkboxradio( "widget" );
```

Comment:
I'm skipping the rest of this file. There's plenty more issues, more of the same.